doc/cli_vs_yum.rst

```diff
@@ -396,9 +397,30 @@ Detailed table for ``package-cleanup`` replacement:
 ``package-cleanup --oldkernels``         ``dnf remove --oldinstallonly``
 ==================================       =====================================
 
-Utilities that have not been ported yet:
+================
+yum-updateonboot
+================
+
+DNF does not have a direct drop-in replacement of yum-updateonboot command.
+However, the similar result can be achieved by using ``dnf automatic`` command (see :doc:`automatic`).
+
+You can either use the shortcut::
+
+  $ systemctl enable dnf-automatic-install.timer && systemctl start dnf-automatic-install.timer
+
+Or set ``apply_updates`` option of ``/etc/dnf/automatic.conf`` to True and use generic timer unit::
+
+  $ systemctl enable dnf-automatic.timer && systemctl start dnf-automatic.timer
+
+The timer is in both cases activated 1 hour after the system was booted up and then repetitively each 24 hours. There is also a random delay on these timers set to 5 minutes. These values can be tweaked via ``dnf-automatic*.timer`` config files located in the ``/usr/lib/systemd/system/`` directory.
+
+None of ``/etc/sysconfig/yum-updateonboot`` configuration options (REBOOT_RPMS, GROUPLIST, OPTIONS) are supported.
```

Comment:
This is unclear.

Reply:
Thanks, I've removed the last sentence. Other issues are fixed.